### DISCUSSION TITLE
My accout was charged $10 dollars thrice but my Github copilot stopped working

### DISCUSSION THREAD

**Discussion by john9384** [community/community#38844] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello does anyone know the GitHub customer care email?
My account was billed three times but yet my GitHub copilot is not working. $10 dollars each. I got a mail saying there was a problem with my payment but I was deducted still. How do I reach out to lay complaints? I followed the contact link in the mail but that brought me here.
Any help would be highly appreciated pls.

**Comment by DSDmark** [comment]
Here is **github-support** go there and create your **Tickets** like that:-
> [**GitHub support**](https://support.github.com/request)
---

![2022-11-11_16-08](https://user-images.githubusercontent.com/85744816/201323226-4a25a839-c152-44c4-a051-926783457b36.png)
